### PR TITLE
Fix/lint issues

### PR DESCRIPTION
**Pull request by pheis**
What was changed?

1. Lint issues were fixed
2. Prettier and eslint were updaed
3. Some linting configs were tweaked

Why was the change made?

Master was borken. Remember to run lint before committing : ). Github actions should also be setupped for this.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ module.exports = {
 
 
     'prettier',
-    'prettier/@typescript-eslint',
-    'prettier/react',
```
Comment (pheis) [inline .eslintrc.js:9]: These are merged into prettier. They are no more.

File: .prettierrc.js [hunk 2]
```diff
@@ -2,4 +2,5 @@ module.exports = {
   printWidth: 80,
   singleQuote: true,
   trailingComma: 'all',
+  arrowParens: 'avoid',
```
Comment (pheis) [inline .prettierrc.js:5]: This was default before, now it must be explicitly configured.

File: package.json [hunk 3]
```diff
@@ -7,9 +7,8 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "npm run lint && jest",
-    "eslint:check": "eslint --print-config ./src/index.ts | eslint-config-prettier-check",
```
Comment (idolgov) [inline package.json:10]: The CLI tool for checking compatible rules show now be run like this:

```
eslint-config-prettier src/index.ts
```

Comment (pheis) [reply package.json:10]: Fixed this. I looked the docs yesterday but was thinking that there is something wrong with this.

**Review by rottabonus — APPROVED** [review]

**Pull request merged**